go.mod

```diff
@@ -5,6 +5,7 @@ go 1.12
 require (
 	cloud.google.com/go v0.39.0
 	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
+	github.com/Microsoft/go-winio v0.4.14
```

Comment:
@pgavlin this is brought in through the proxy_windows.go file.  does it need ot somehow be gated in the module file as well? i.e. does it need some sort of `// +build windows` thingy here?

Reply:
Nope, this is fine.